db/dcrpg/pgblockchain_charts_test.go

```diff
@@ -72,26 +39,45 @@ func TestMain(m *testing.M) {
 // It also checks the order and duplicates in the x-axis dataset.
 func TestPgCharts(t *testing.T) {
 	charts := cache.NewChartData(context.Background(), 0, &chaincfg.MainNetParams)
-	charts.DiffInterval = 10
```

Comment:
Setting this is no longer OK since `DiffInterval` and other `chaincfg.Params` fields are used in the Windows data sets.

Reply:
Ah, I recall this now.   I though this was causing trouble in some of the charts updates, but perhaps that's not an issue now with the other fixes.  `chainParams.StakeDiffWindowSize` was being used inconsistent with the fudged DiffInterval somewhere.  I'll try to sort that out.